sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/RequestResponseChannel.java

```diff
@@ -156,7 +156,7 @@ protected RequestResponseChannel(AmqpConnection amqpConnection, String connectio
                 handleError(error, "Error in ReceiveLinkHandler.");
                 onTerminalState("ReceiveLinkHandler");
             }, () -> {
-                closeAsync("ReceiveLinkHandler. Endpoint states complete.").subscribe();
```

Comment:
Did a merge go wrong? This PR shouldn't include any changes to Azure Core.

Reply:
file reverted to master